junit-platform-console/src/main/java/org/junit/platform/console/options/AvailableOptions.java

```diff
@@ -240,6 +240,24 @@ static class FilterOptions {
 		@Option(names = { "-exclude-package" }, arity = "1", hidden = true)
 		private List<String> excludePackages2 = new ArrayList<>();
 
+		@Option(names = {
+				"--include-methodname" }, paramLabel = "PATTERN", arity = "1", description = "Provide a regular expression to include only methods whose fully qualified names without parameters match. "
+						+ //
+						"When this option is repeated, all patterns will be combined using OR semantics.")
+		private List<String> includeMethodNamePatterns = new ArrayList<>();
+
+		@Option(names = { "-include-methodname" }, arity = "1", hidden = true)
+		private List<String> includeMethodNamePatterns2 = new ArrayList<>();
```

Comment:
The single dash options only exist for backwards compatibility and should not be added for new options, i.e. here and below for `-exclude-methodname`.

Reply:
Sorry for the late reply, I will find time to rework this PR recently.